go-packages/meep-couch/db.go

```diff
@@ -56,12 +67,18 @@ func NewConnector(addr string, dbName string) (rc *Connector, err error) {
 			c.addr = addr
 		}
 
-		var address []string = strings.SplitAfter(addr, "http://")
-		var newAddr = address[0] + "admin:admin@" + address[1]
-		c.dbClient, err = kivik.New(context.TODO(), "couch", newAddr)
+		c.dbClient, err = kivik.New(context.TODO(), "couch", addr)
 		if err != nil {
 			return nil, err
 		}
+
+		err = c.dbClient.Authenticate(context.TODO(), &chttp.CookieAuth{Username: "admin", Password: "admin"})
```

Comment:
We should pass these credentials using an environment variable instead. Please create a new task for this that we can address later. For now, we can leave it as is.

Reply:
ok